### PR TITLE
Remove i want to participate

### PR DESCRIPTION
**Pull request by michalmasrna1**
Closes: #166

### CODE REVIEW

**Review by Matushl — COMMENTED** [review]
Je to fajn, len zmen to cislo na boolean  a iny nazov a rebasni na master nech tam je ta zmena od mihala.

Kludne ako dalsi commit rozdel subor problems a problem.

File: src/components/Problems/Problems.tsx [hunk 1]
```diff
@@ -156,8 +184,31 @@ export const Problems: FC = () => {
 
   const {hasPermissions, permissionsIsLoading} = useHasPermissions()
 
+  const [deleteDialogId, setDeleteDialogId] = useState<number | undefined>()
+  const close = () => setDeleteDialogId(undefined)
```
[the remark this answers is not in the document]
Comment (michalmasrna1) [reply src/components/Problems/Problems.tsx]: fixed

**Comment by kovacspe** [comment]
Closes: #166

**Review by Matushl — APPROVED** [review]
LGTM

**Pull request merged**